packages/dev/docs/pages/react-aria/why.mdx

```diff
@@ -0,0 +1,93 @@
+their design system to match their brand. There are very few built-in UI controls in the browser, and those
+that do exist are very hard to style. This leads to web developers at every company needing to rebuild every 
+control from scratch. This represents **millions of dollars** of investment for each company to **duplicate
+work** that many other companies are also doing.
+
+The fully styleable primitives that the web offers (e.g. `<div>`) are quite powerful, but they lack semantic
+meaning. This means that **accessibility is often missing** because assistive technology cannot make sense of the
+div soup that we use to implement our components. Even if you use [ARIA](https://www.w3.org/TR/wai-aria-1.2/)
+to provide semantics, you still need to implement all of the behavior for each component from scratch
+using JavaScript, and this can be tricky to do across many devices and interaction models.
+
+Unfortunately, many companies and teams don't have the resources or time to prioritize features like
+accessibility, internationalization, full keyboard navigation, touch interactions, and more.
+This leads to many web apps having sub-par accessibility and interactions, and contributes to the
+perception of the web as an inferior app platform compared to native apps.
+
+## React Aria
+
+React Aria separates the behavior and accessibility implementation for many common UI components out into
+[React Hooks](https://reactjs.org/docs/hooks-intro.html), which enables them to be reused easily between
+design systems. You remain in complete control over the rendering and styling aspects of your components,
+but get most of the [behavior](interactions.html), [accessibility](accessibility.html), and
+[internationalization](internationalization.html) support for free.
+
+Rather than starting with only divs and building everything yourself, you start with elements that have semantic meaning,
+behavior, and interactions built in. This allows you to **build components much more quickly**, and ensures that they work well
+across mouse, touch, and keyboard interactions, as well as with screen readers and other assistive technology.
+
+## Customizability
+
+The main reason developers need to create their own component libraries from scratch is styling. Built-in browser UI controls
+are not customizable enough, and in many cases do not exist at all. Given this, React Aria is intentionally very low level:
+unlike many open source component libraries, it provides **no rendering at all**, just behavior and accessibility.
+We impose no specific DOM structure or styling methodology. You can use CSS classes, CSS-in-JS, etc.
+The hooks are small and **composable**, which allows you to combine them together and achieve your design with only the 
+features you need in your component.
+
+In terms of behavior, most components have behavior that is very well understood. The 
```

Comment:
In terms of behavior, the 
[WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.2/) published ...
OR
Most components have behavior that is consistent even across design systems. The WAI-ARIA....